components/econet/econet.cpp

```diff
@@ -392,13 +393,7 @@ void Econet::loop() {
     return;
   }
 
-  // Quickly send writes or reads from the Home Assistant service but delay regular reads.
-  if (!pending_writes_.empty() || !datapoint_ids_for_read_service_.empty() ||
-      (now - this->last_request_ > this->update_interval_millis_ / request_mods_)) {
```

Comment:
Scary... I had the same thoughts and had already written up this change as you described. It might be implemented slightly different than you suggest but let's review this when I get the next commit up.